config/settings.yml

```diff
@@ -442,3 +442,13 @@ va_mobile:
   mock: false
   key_path: /fake/client/key/path
   timeout: 15
+
+vbms:
+  url: "https://internal-dsva-vagov-dev-fwdproxy-1893365470.us-gov-west-1.elb.amazonaws.com:4449"
+  saml: "vetsapi.client.vbms.aide.oit.va.gov.saml-token.xml"
+  keypass: "{{ lookup('credstash', 'vbms.dev.key_password') }}"
+  ca_cert: VBMS-Client-Signing-CA.crt
```

Comment:
if these certs are fake then I would include the work fake somewhere in the title so that anyone looking at this won't think we somehow accidentally exposed a cert without context.

Reply:
These are the names of real certs, but not the certs themselves. They are dropped on the box via the ansible deploy script. What I believe @ATeal was referring to was the fake certs in the spec fixtures folder.

Reply:
^ Yes I was